lib/main.js

```diff
@@ -121,6 +121,12 @@ function initSession(shiny, options, shinyServer) {
 
       factory(url, ctx, (err, conn) => {
         pc.resolve(err, conn);
+        conn.onclose = e => {
```

Comment:
Prior to this, there are a bunch of event handlers registered for reconnect/disconnect events. Should those still happen? Staying with the style of the rest of this project, it feels like a different decorator should be used when reconnects are not supported and the `showDisconnected` should continue to respect a disconnect event.